models/value_set.go

```diff
@@ -49,61 +49,60 @@ type CodeSet struct {
 	Values []Concept
 }
 
-func (v ValueSetMap) CodeDisplayWithPreferredCodeForField(entry *Entry, coded *Coded, MapDataCriteria Mdc, codeType string, field string) CodeDisplay {
+func (v ValueSetMap) GenerateCodeDisplay(entry *Entry, coded *Coded, MapDataCriteria Mdc, codeType string) CodeDisplay {
 	codeDisplay, err := entry.GetCodeDisplay(codeType)
 	if err != nil {
-		log.Fatalln(err)
+		log.Fatal(err)
```

Comment:
Does the template not execute if its passed in argument is `nil`? I was under the impression that it does execute, but `.` is set to `nil`, which is dangerous unless we change the templates.

But if that were the case, would it not make sense to simply return an empty `CodeDisplay`?